ocsmesh/raster.py

```diff
@@ -1427,7 +1428,7 @@ def _get_raster_contour_single_window(
             for path in path_collection.get_paths():
                 try:
                     features.append(LineString(path.vertices))
-                except ValueError:
+                except (ValueError, GEOSException):
                     # LineStrings must have at least 2 coordinate tuples
                     pass
```

Comment:
Let's modify this code to just check for the length of coordinates instead, that way there won't be any need to import the new exception. Also it would be really nice if you could add a very simple test case that demonstrates this.

Reply:
Funny, I had that as a first try but thought to do a pythonic way :)

I'm working on the test. I'm installing a venv rather than conda.